src/cpp/CMakeLists.txt

```diff
@@ -415,7 +415,11 @@ if(UNIX)
    find_library(DL_LIB "dl")
    find_library(SQLITE_LIB "sqlite3")
    get_filename_component(SQLITE_LIB "${SQLITE_LIB}" REALPATH)
-   find_library(PQ_LIB "pq")
+   if (APPLE)
+      set(PQ_LIB "/usr/local/lib/libpq.dylib")
```

Comment:
@gtritchie Also if you don't mind let's put a comment here about why we're specifically setting the path on OSX (qt bundle script will only bundle dylibs, so if this is pointing to an xcode library spec file it won't get copied into the bundle)

Reply:
Good idea, done.